ui/app/components/job-status/allocation-status-block.hbs

```diff
@@ -39,6 +39,8 @@
           <span class="alloc-health-indicator">
             {{#if (eq @health "healthy")}}
               <FlightIcon @name="check" @color="#25ba81" />
+            {{else if (eq @health "unhealthy")}}
+              <FlightIcon @name="x" @color="#c84034" />
```

Comment:
This is the little top-right-corner status indicator that shows up when enough of an alloc exist to summarize it in a "+N" block